frontend/src/components/Tiers.js

```diff
@@ -1,128 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
Did you do a search for this everywhere? Looks like we are still using `Tiers` on the DonatePage.js in containers/